CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Verison 0.5.0 [unreleased]
 WIP
 
 - [API] Added `API docs <https://github.com/openwisp/openwisp-network-topology/#rest-api>`_
+- [API] Added ` API authentication and permissions <https://github.com/openwisp/openwisp-network-topology#openwisp-network-topology-api-auth-required>`_
```

Comment:
there's an extra space: ``` API authentication`` > ```API authentication``

A more informative change log line would be:

- [change] **Backward incompatible**: API and visualizer views now require authentication by default.
  This can be changed through the new [OPENWISP_NETWORK_TOPOLOGY_API_AUTH_REQUIRED](https://github.com/openwisp/openwisp-network-topology#openwisp-network-topology-api-auth-required) setting